app/views/shared/_landing_nav.html.erb

```diff
@@ -0,0 +1,34 @@
+<div id="landing_nav">
+  <div class="flex flex-row justify-between pt-10 px-28 w-full">
+    <a href="/">
+      <img src="<%= asset_path("icon/Logo.svg") %>" alt="DPI Logo" class="h-[2.5rem]">
+    </a>
+    <div class="bg-white opacity-70 flex items-center	justify-around border-slate-300 rounded-2xl w-[26rem] h-[3.2rem]">
+      <ul class="flex gap-10 text-s font-normal text-black max-w-[26]">
+        <li><a href="/" class="hover:text-gray-600 font-semibold">Home</a></li>
+        <li><a href="/" class="hover:text-gray-600">Demo</a></li>
+        <li><a href="/" class="hover:text-gray-600">Dashboard</a></li>
+      </ul>
+    </div>
+    <% if user_signed_in? %>
+      <div class="flex gap-4 items-center">
+        <div class="text-s text-[#676666] font-normal">
+          <%= link_to "Edit", edit_user_registration_path %>
+        </div>
+        <div class="text-s font-medium bg-[#202020] text-[#FFFFFF] border border-transparent rounded-md
+             py-2 px-5 mr-auto hover:bg-transparent hover:text-[#202020] hover:border-[#202020]">
+          <%= button_to "Sign out", destroy_user_session_path, method: :delete, data: { turbo: false } %>
+        </div>
+      </div>
+    <% else %>
+      <div class="flex gap-4 items-center">
+        <div class="text-s text-[#676666]">
+          <%= link_to "Sign Up", new_user_registration_path %>
+        </div>
+        <%= link_to "Sign In", new_user_session_path, class: "text-s font-medium bg-[#202020] text-[#FFFFFF] border border-transparent rounded-md
```

Comment:
What does the nave look like when a user is signed in?

Reply:
If user signed in they see Edit and Sign Out